scripts/camViewer

```diff
@@ -270,6 +269,20 @@ do
 
 done
 
+# Skip verify-hutch if this is the remote SSH session
+if [ -n "$IS_REMOTE_SSH" ]; then
```

Comment:
I believe this variable is no longer used, there are a couple references to it remaining